en/developer/reference/articles/02-workflow/05-notification-templates.markdown

```diff
@@ -0,0 +1,287 @@
+| ${passwordPolicyPermission}  |          | &#10004;  | [PasswordPolicyPermission](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/service/permission/PasswordPolicyPermission.html) |
+| ${expandoColumnLocalService} |          | &#10004;  | [ExpandoColumnLocalService](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/expando/kernel/service/ExpandoColumnLocalService.html) |
+| ${taskComments}              | &#10004; |           | The comments, if any, left by the reviewer |
+| ${staticFieldGetter}         |          | &#10004;  | [StaticFieldGetter](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/util/StaticFieldGetter.html) |
+| ${htmlUtil}                  |          | &#10004;  | [HtmlUtil](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/util/HtmlUtil.html) |
+| ${languageUtil}              |          | &#10004;  | [LanguageUtil](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/language/LanguageUtil.html) |
+| ${enumUtil}                  | &#10004; |           | DESCRIBE |
+| ${windowStateFactory}        |          | &#10004;  | [WindowStateFactory](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/portlet/WindowStateFactory.html) |
+| ${companyId}                 | &#10004; |           | The ID of the portal's Company/Virtual Instance |
+| ${unicodeFormatter}          |          | &#10004;  | [UnicodeFormatter](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/util/UnicodeFormatter.html) |
+| ${propsUtil}                 |          | &#10004;  | [PropsUtil](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/util/PropsUtil.html) |
+| ${browserSniffer}            |          | &#10004;  | [BrowserSniffer](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/servlet/BrowserSniffer.html) |
+| ${portletProviderAction}     | &#10004; |           | Returns the outcome of the [PortletProvider#Action](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/portlet/PortletProvider.Action.html) call |
+| ${taskName}                  | &#10004; |           | The name of the task the notification is in |
+| ${httpUtil}                  |          | &#10004;  | [HttpUtil](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/util/HttpUtil.html) |
+| ${portletURLFactory}         |          | &#10004;  | [PortletURLFactory](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/portlet/PortletURLFactory.html) |
+| ${imageToken}                | &#10004; |           | DESCRIBE |
```

Comment:
@rafaprax another one i could use help with :point_up_2:

Reply:
It is an instance of [WebServerServletToken](https://docs.liferay.com/dxp/portal/7.2-latest/javadocs/portal-kernel/com/liferay/portal/kernel/webserver/WebServerServletToken.html). I took a look and it is use just to generate a token based on an imageId or something like this. I don't know what the use of that is.